views/submit-payment.ejs

```diff
@@ -35,9 +35,7 @@
         <form action="<%= next_url %>">
 
             <div class="form-group">
-
-                <button type="submit" class="govuk-button" data-module="govuk-button">Pay</button>
-
+                <button type="submit" class="govuk-button" data-module="govuk-button" data-testid="app-id" data-value="<%= applicationId %>" >Pay</button>
```

Comment:
The addition of this extra data is to help with automated accessibility testing. I know this sounds a but vague but I can create a video to explain it more if you want.